documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -135,7 +135,16 @@ We request OpenAPI(Swagger) spec authoring be assigned to engineers who have an
 | [R2007](#r2007) | [LongRunningOperationsWithLongRunningExtension](#r2007) | ARM OpenAPI(swagger) specs |
 | [R2029](#r2029) | [PageableOperation](#r2029) | ARM and Data plane OpenAPI(swagger) specs |
 | [R4006](#r4006) | [DeprecatedXmsCodeGenerationSetting](#r4006) | ARM and Data plane OpenAPI(swagger) specs |
-  
+
+
+### RPaaS Violations
+
+#### RPaaS Errors
+
+| Id | Rule Name | Applies to |
+| --- | --- | --- |
+| [R4014](#r4014) | [LongRunningOperationMustSupport201](#r4014) | ARM OpenAPI(swagger) specs |
+
```

Comment:
Is this rule similar as R2005 ?

Reply:
No. This is RPaaS specific and validates only PUT. I agree there is a bit of overlap.
R2005 is actually not correct. Is this rule active ?
Even 202 is allowed long running status code for PUT, DELETE & POST. 

_**Description**: The allowed response status codes for a long DELETE operation are "200", "204". The allowed response status codes for a POST operation are "200", "201" & "204". The allowed response status codes for a PUT operation are  "200" & "201"._


_**R4014 (RPaaS specific rule)**_ 
- Only 201 is supported
-x-ms-long-running-operation added (lot of RPs miss this today)
-  x-ms-long-running-operation-options is specified

Reply:
also updated to use R4023

Reply:
you may want to update R2005 to include 202 as well. It is not checking x-ms-* stuff. it is just validating status code.

Reply:
Also similar as R2010 which is used for validating POST operation, but make sense